Python/Sorting_Techniques.py

```diff
@@ -0,0 +1,127 @@
+#  Github username : Prathamesh-510
+#  Here I created a simple python file in which all sorting algorithms in the python are discussed.
+
+
+# Function for Bubble Sort
+def bubble_sort(arr):
+    n = len(arr)
+    comparisons = 0
+    swaps = 0
+    for i in range(n):
+        for j in range(0, n - i - 1):
+            comparisons += 1
+            if arr[j] > arr[j + 1]:
+                swaps += 1
+                arr[j], arr[j + 1] = arr[j + 1], arr[j]
+    return comparisons, swaps
```

Comment:
def bubbleSort( sorti ):
    n = len( sorti )

    for i in range( n - 1 ) :
        flag = 0

        for j in range(n - 1) :
            
            if sorti[j] > sorti[j + 1] : 
                tmp = sorti[j]
                sorti[j] = theSeq[j + 1]
                sorti[j + 1] = tmp
                flag = 1

        if flag == 0:
            break

    return sorti